cmd/kube-apiserver/app/server.go

```diff
@@ -418,6 +419,16 @@ func buildGenericConfig(
 		return
 	}
 
+	if genericConfig.EgressSelector != nil {
+		networkContext := egressselector.Cluster.AsNetworkContext()
+		dialer, lastErr := genericConfig.EgressSelector.Lookup(networkContext)
```

Comment:
FYI https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/84381/pull-kubernetes-verify/1192964936888750080

`lastErr` is being defined a second time on this line. Renaming the variable should solve it.

```
# k8s.io/kubernetes/cmd/kube-apiserver/app
cmd/kube-apiserver/app/server.go:426:4: lastErr is shadowed during return
!!! [1109 00:50:48] Call tree:
!!! [1109 00:50:48]  1: /home/prow/go/src/k8s.io/kubernetes/hack/lib/golang.sh:714 kube::golang::build_some_binaries(...)
!!! [1109 00:50:48]  2: /home/prow/go/src/k8s.io/kubernetes/hack/lib/golang.sh:853 kube::golang::build_binaries_for_platform(...)
!!! [1109 00:50:48]  3: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [1109 00:50:48] Call tree:
!!! [1109 00:50:48]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [1109 00:50:48] Call tree:
!!! [1109 00:50:48]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
make[1]: *** [Makefile:93: all] Error 1
```

Reply:
Ah indeed 😅 should be fixed then